core/idrepo/logic/src/main/java/org/apache/syncope/core/logic/ReportLogic.java

```diff
@@ -272,8 +274,8 @@ public void exportExecutionResult(
             final ReportExecExportFormat format) {
 
         // streaming SAX handler from a compressed byte array stream
-        try (ByteArrayInputStream bais = new ByteArrayInputStream(reportExec.getExecResult());
-                ZipInputStream zis = new ZipInputStream(bais)) {
+        try ( ByteArrayInputStream bais = new ByteArrayInputStream(reportExec.getExecResult());  ZipInputStream zis =
```

Comment:
Please revert this formatting-only change.
This will also break checkstyle AFAICT.